docs/sources/project/create-pr.md

```diff
@@ -56,28 +60,35 @@ Always rebase and squash your commits before making a pull request.
         pick 1a79f55 Tweak some of the other text for grammar
         pick 53e4983 Fix a link
         pick 3ce07bb Add a new line about RHEL
-        
-    If you run into trouble, `git --rebase abort` removes any changes and gets
-    you back to where you started. 
 
-4. Squash the `pick` keyword with `squash` on all but the first commit.
+    If you are not sure about the rebase at this point, delete all the lines and
+    quit from the editor which aborts the rebase process.
+        
+5. Replace the `pick` keyword with `squash` on all but the first commit.
 
         pick 1a79f55 Tweak some of the other text for grammar
         squash 53e4983 Fix a link
         squash 3ce07bb Add a new line about RHEL
 
-    After closing the file, `git` opens your editor again to edit the commit
-    message. 
+    Save the changes and quit from editor at which point, `git` will start the actual 
+    rebase process. 
+
+6. If there are conflicts, you need to resolve them. `git status` will show you
+    which files, if any, are in conflict. You need to manually resolve conflicts in 
+    each file and then stage it using the command `git add`. 
+
+    If you are not sure about the rebase, `git rebase --abort` at this point removes 
```

Comment:
at this point -> at *any* point? Could make a difference, and make people more confident.